docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,366 @@
+# Getting Started
+This document is intended as a "getting started" guide. The audience are developers looking to modify or *publish* the packages to the universe. This document is written in tutorial and walk-through format. The goal is to help you "get started". It does not go into great depth on some of the conceptual or inner details. This guide aims at making a user familiar with the concepts of what a Package is and what are the roles of marathon and universe in the package life cycle.
+
+
+## Prerequisites
+Before the launch, make sure you have met the following conditions.
```

Comment:
Was going for the rocketship flavored pun here, guess it wasn't all clear! Updated the description.